client/lib/route/path.ts

```diff
@@ -102,17 +103,20 @@ export function sectionify( path, siteFragment ) {
 	return untrailingslashit( basePath );
 }
 
-export function getStatsDefaultSitePage( slug ) {
+export function getStatsDefaultSitePage( slug?: SiteId | SiteSlug ): URLString {
```

Comment:
`slug` is a funny name when you put the type next to it, but that's fine 🙂

Reply:
Renamed to `siteFragment`, to match the other instances 👍